buildtools/webpack/plugins.js

```diff
@@ -5,9 +5,17 @@ const { readFileSync } = require('fs');
 const { join } = require('path');
 const { DefinePlugin, BannerPlugin, IgnorePlugin, optimize } = require('webpack');
 const { some } = require('underscore');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 
 const UglifyJsPlugin = optimize.UglifyJsPlugin;
 
+function webpackBundleAnalyzer() {
+  return new BundleAnalyzerPlugin({
+    openAnalyzer: false,
+    reportFilename: 'okta-sign-in.html',
```

Comment:
Can we change the filename to be more specific?

Reply:
+1

Reply:
`okta-sign-in-bundle-analyzer-result.html` ?

Reply:
We probably don't event need to namespace it, unless we for-see multiple analysis files:

```
bundle-analyzer-result.html
```

Reply:
keep the namespace for now :)